### ISSUE TITLE
streamz??

### ISSUE THREAD

**Issue by martindurant** [holoviz/lumen#66]
I'm *sure* we talked about this already, but it came up while chatting with @seibert .

In this library, you have data sources which poll a REST endpoint on some regular period, accumulate data, turn into dataframes, do some processing, and finally update a plot in the frontend (some part of a dashboard). Streamz does all of these things, including the ability to create multiple different views of the same dataset on one or more plots. Why not use streamz?

Would it make any difference if Intake already supported streamz? This is something I have long been meaning to do, but just needed the spur ( https://github.com/intake/intake-streamz very incomplete).

**Comment by philippjfr** [comment]
Absolutely, in fact I'd like to generalize Lumen to support a variety of datatypes including xarray datasets. Still considering the best approach here to ensure all the filtering logic and so one works across all of them. Building out intake-streamz would certainly help too.

**Comment by martindurant** [comment]
Intake-streamz is on pypi and awaiting conda-forge at https://github.com/conda-forge/staged-recipes/pull/13692
